### PR TITLE
update short for tool import

### PR DESCRIPTION
**Pull request by daixiang0**
In fact, it does not list function.

Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: cmd/tk/tool.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func jpathCmd() *cobra.Command {
 func importsCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "imports [file]",
-		Short: "list all transitive imports of a file",
+		Short: "import a transitive file",
```
Comment (sh0rez) [inline cmd/tk/tool.go]: Huh, I do think the old text was correct.

This tool goes ahead prints every direct and recursive import of the file to have a flat list of all dependencies.

So after all, it *lists all transitive / recursive imports of a file*

Comment (daixiang0) [reply cmd/tk/tool.go]: Seems that i misunderstand it with `ls` in shell, so the file param is still not optional?

Comment (sh0rez) [reply cmd/tk/tool.go]: No, because it creates a list (an array) of all recursive imports of _that specific_ file

**Comment by sh0rez** [comment]
Can we close this, because the current description is correct?

**Comment by daixiang0** [comment]
Still has question, since file param must need, use "[]" is not good.

File: cmd/tk/tool.go (hunk 1 above)
Comment (daixiang0) [reply cmd/tk/tool.go]: so why not named as "tool list"?

**Comment by sh0rez** [comment]
> Still has question, since file param must need, use "[]" is not good since it means optional for most case.

Aah, I see your point. I wasn't aware of the fact that the bracket syntax in CLI help means *optional*. Yes, the argument is definitely required. Is there place that documents a canonical syntax for CLI's?

Would it be sufficient to remove the brackets change it to 
```go
  Use:  "imports FILE",
```

**Comment by daixiang0** [comment]
"Command description syntax" chapter in https://en.wikipedia.org/wiki/Command-line_interface may help you.

**Comment by sh0rez** [comment]
Oh nice! Thank you :) Tanka definitely does not accord to these guidelines, opened an issue for this

**Comment by daixiang0** [comment]
update done.

**Comment by sh0rez** [comment]
Closing in favor of #94

**Pull request closed without merging**